src/python/library/requirements/requirements_grpc.txt

```diff
@@ -27,7 +27,7 @@
 # There is memory leak in later Python GRPC (1.43.0 to be specific),
 # use known working version until the memory leak is resolved in the future
 # (see https://github.com/grpc/grpc/issues/28513)
-grpcio>=1.41.0
+grpcio>=1.41.0,<=1.59.3
```

Comment:
can we explain why `<=1.59.3` is added in the comments above, like the `>=1.41.0` requirement is added?